### PR TITLE
[TASK][27014] Submit job

### PR DESCRIPTION
**Pull request by thuyptsun**
https://edu-redmine.sun-asterisk.vn/issues/27014
Screenshot from 2020-08-24 09-11-21
Screenshot from 2020-08-24 09-11-50
Screenshot from 2020-08-24 09-13-18

### CODE REVIEW

**Review by vinhcnt — APPROVED** [review]

File: app/views/job_posts/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+              <div class="row">
+                <div class="col-md-12">
+                  <div class="form-group app-label mt-2">
+                    <%= f.label :title do %>
+                      <div class="text-muted">
+                        <%= t ".title2" %><span class="text-danger"> *</span>
+                      </div>
+                    <% end %>
+                    <%= f.text_field :title, class: "form-control resume", required: true %>
+                    <div class="invalid-feedback" style="display: block;">
```
Comment (hieplt-0914) [inline app/views/job_posts/new.html.erb]: cho vào file css

File: app/views/job_posts/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                  </div>
+                </div>
+                <div class="col-md-12">
+                  <div class="form-group app-label mt-2">
+                    <%= f.label :salary_to do %>
+                      <div class="text-muted">
+                        <%= t ".salary_to" %><span class="text-danger"> *</span>
+                      </div>
+                    <% end %>
+                    <%= f.number_field :salary_to, min: 0, class: "form-control", required: true %>
```
Comment (hieplt-0914) [inline app/views/job_posts/new.html.erb]: Settigns

Comment (hieplt-0914) [reply app/views/job_posts/new.html.erb]: Thêm max

File: app/views/job_posts/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                  </div>
+                </div>
+                <div class="col-md-12">
+                  <div class="form-group app-label mt-2">
+                    <%= f.label :location do %>
+                      <div class="text-muted">
+                        <%= t ".location" %><span class="text-danger"> *</span>
+                      </div>
+                    <% end %>
+                    <%= f.text_field :location, class: "form-control", required: true %>
```
Comment (hieplt-0914) [inline app/views/job_posts/new.html.erb]: thêm validate ký tự ở frontend

**Pull request merged**